plugins/SegmentEditor/tests/Integration/SegmentEditorTest.php

```diff
@@ -191,7 +191,7 @@ public function testDeleteSegment()
         $idSegment2 = API::getInstance()->add('name 2', 'searches>1,visitIp!=127.0.0.1', $idSite = 1, $autoArchive = 1, $enabledAllUsers = 1);
 
         $deleted = API::getInstance()->delete($idSegment2);
```

Comment:
```suggestion
        API::getInstance()->delete($idSegment2);
```

No need to keep the unused variable around from the changed `void` return.